src/Weather.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class Weather extends Component {
+  state = {
+    temp: "",
+    pic: "",
+    description: "",
+  };
+
+  componentDidMount() {
+    fetch(
+      `http://api.weatherstack.com/current?access_key=${process.env.REACT_APP_WEATHER}&query=Toronto&units=m`
+    )
+      .then((res) => {
+        return res.json();
+      })
+      .then((data) =>
+        this.setState({
+          pic: data.current.weather_icons[0],
+          description: data.current.weather_descriptions[0],
+          temp: data.current.temperature,
+        })
+      )
+      .catch((err) => {
+        console.log(err);
```

Comment:
Consider displaying some placeholder text ('Unable to retrieve weather data') to let your user know what's going on and handle the error more gracefully.